pkg/collector/scheduler.go

```diff
@@ -134,10 +140,22 @@ func (s *CheckScheduler) getChecks(config integration.Config) ([]check.Check, er
 	checks := []check.Check{}
 	numLoaders := len(s.loaders)
 
+	initConfig := commonInitConfig{}
+	err := yaml.Unmarshal(config.InitConfig, &initConfig)
```

Comment:
if `loader` is not defined it will return an empty string?

Reply:
Yes, when not defined, it defaults to the "zero value" of the string type (empty string `""`).